.github/workflows/image.yaml

```diff
@@ -29,6 +29,48 @@ on:
       - release-*
 
 jobs:
+  gh-runner-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        name: Check out code
+      - name: Calculate build vars
+        id: vars
+        run: |
+          if [ "${{github.event_name == 'pull_request'}}" = "true" ]; then
+            echo "IMAGE_PUSH=false" >> $GITHUB_ENV
+            echo "IMAGE_OUTPUTS=type=docker,dest=/tmp/k8s-test-infra-image.tar" >> $GITHUB_ENV
+            echo "IMAGE_PLATFORMS=linux/amd64" >> $GITHUB_ENV
+          else
+            echo "IMAGE_PUSH=true" >> $GITHUB_ENV
+            echo "IMAGE_OUTPUTS=type=registry,push=true" >> $GITHUB_ENV
+            echo "IMAGE_PLATFORMS=linux/amd64,linux/arm64" >> $GITHUB_ENV
+          fi
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ghcr.io/nvidia/k8s-test-infra
+      - name: Build and push
+        uses: docker/build-push-action@v6
+        with:
+          push: ${{ env.IMAGE_PUSH }}
+          context: .
+          outputs: ${{ env.IMAGE_OUTPUTS }}
+          platforms: ${{ env.IMAGE_PLATFORMS }}
+          tags: ghcr.io/nvidia/k8s-test-infra:gh-runner
```

Comment:
Are we interested in the version at all?

Reply:
I was thinking that no, since we want latest versions from both holodeck and the runner.

Reply:
how about now?

Reply:
I like how it reads now